src/main/java/com/leaseforlove/tagsmanagementservice/infraestructure/storage/migrations/config/ClientInitializerChange.java

```diff
@@ -0,0 +1,25 @@
+package com.leaseforlove.tagsmanagementservice.infraestructure.storage.migrations.config;
+
+import io.mongock.api.annotations.ChangeUnit;
+import io.mongock.api.annotations.Execution;
+import io.mongock.api.annotations.RollbackExecution;
+import org.springframework.data.mongodb.core.MongoTemplate;
+
+
+@ChangeUnit(id="client-initializer", order = "001", author = "mongock")
```

Comment:
aqui no author, coloca o nome de usuario do github, os ids, acho que vale a pena colocar a data.
essa primeira migration, pode ser para criar nossa collection(database). lembro vagamente do order ser opcional, preciso confirmar, mas se for, podemos revisitar a necessidade dele